src/modules/Popup/Popup.js

```diff
@@ -228,7 +228,7 @@ export default class Popup extends Component {
 
     // Lets detect if the popup is out of the viewport and adjust
     // the position accordingly
-    const positions = _.without(POSITIONS, position)
+    const positions = _.without(POSITIONS, position).push(position)
```

Comment:
This line doesn't make sense, you remove `position` from array and then push it again :confused: